src/arrays/programs_involving_multidimensional_data/hard/q6/MaxVal.java

```diff
@@ -0,0 +1,27 @@
+package arrays.programs_involving_multidimensional_data.hard.q6;
+
+/*
+Write a Java program that initializes a 2D array of random integers of size 5 by 5. All random values generated will be
```

Comment:
We should add a method component to this question so we can make sure they are actually using an array (probably finding the maximum value).